R/step-call-pivot_wider.R

```diff
@@ -128,26 +128,13 @@ pivot_wider.dtplyr_step <- function(data,
   }
 
   if (!is.null(names_glue)) {
-    out <- step_call(
-      out,
-      "setnames",
-      args = list(old = new_vars, new = glue_vars),
-      vars = c(id_cols, glue_vars),
-      in_place = FALSE
-    )
+    out <- step_setnames(out, !!!stats::setNames(new_vars, glue_vars), in_place = FALSE)
```

Comment:
Do you think it would be easier to understand if we turned `new_vars` and `old_vars` into explicit arguments?